protocol/accessibility/README.md

```diff
@@ -0,0 +1,93 @@
+Accessibility
+=============
+
+A guide for auditing and maintaining accessible web sites and apps.
+
+Basics
+------
+
+thoughtbot strives for AA level [Web Content Accessibility Guideline (WCAG)][wcag] compliance. Perform one or more of these checks to ensure your work is accessible. 
+
+### Automation
+
+Automated checks can catch a lot of common issues before they reach production.
+
+* Prefer running audits on your local build using a tool such as [WAVE][wave] or the [axe Chrome extension][axe-chrome]
```

Comment:
"Prefer" -- "indicates a better option and its alternative to watch out for", says the README -- makes me think that we should prefer these tools over other tools. Maybe "Use" makes more sense for most of the accessibility guide?